### DISCUSSION TITLE
Define Bellmanford algorithm.

### DISCUSSION THREAD

**Discussion by ankit-akash** [ankit-akash/DSA#44] [Q&A] [answered]
brief

**Comment by fcasamento** [comment, marked as answer]
Dijkstra’s algorithm is a Greedy algorithm and the time complexity is O((V+E)LogV) (with the use of the Fibonacci heap). Dijkstra doesn’t work for Graphs with negative weights, Bellman-Ford works for such graphs. Bellman-Ford is also simpler than Dijkstra and suites well for distributed systems. But time complexity of Bellman-Ford is O(V * E), which is more than Dijkstra.

![1_qNNJpdY-o9oIh9Yg4O-d-g](https://user-images.githubusercontent.com/19937845/205523326-bd8577a3-a935-42f3-9d3c-4c1d386f3cdc.gif)

**Reply by fcasamento** [reply]
@ankit-akash Mark this as an answer as well, we can help each other. 🙏🏻 Pls
